ts/client/src/client.ts

```diff
@@ -2979,40 +2948,43 @@ export class MangoClient {
     );
   }
 
-  public buildHealthRemainingAccounts(
```

Comment:
remove public, code is not used anywhere else in client, and in ui, if some npm client user complains, I will fix it on demand